### PR TITLE
Use early returns consistently

### PR DESCRIPTION
**Pull request by danthedaniel**
Just a code style change.

### CODE REVIEW

**Comment by creesch** [comment]
Let me think about this. Not sure if I prefer it

**Comment by creesch** [comment]
Also has a merge conflict now due to the whitespace changes. 

Anyway, I don't mind early returns. But, I would like them to just be wrapped in curly brackets for consistency.  One of the benefits of early returns is to reduce cognitive load and improve readability. For me having consistent use of curly brackets helps in that aspect well so that if statements always follow the same pattern.

**Comment by danthedaniel** [comment]
@creesch good idea. Also went through and made that consistent in the JS code.

File: src/client/java/dev/creesch/WebchatClient.java [hunk 1]
```diff
@@ -22,51 +22,60 @@ public void onInitializeClient() {
...
-                LOGGER.info("Got chat message as JSON: {}", json);
-                webInterface.broadcastMessage(json);
+            if (client.world == null) {
+                return;
             }
+
+            String json = Text.Serialization.toJsonString(message, client.world.getRegistryManager());
+            LOGGER.info("Got chat message as JSON: {}", json);
+            webInterface.broadcastMessage(json);
+        });
```
Comment (creesch) [inline src/client/java/dev/creesch/WebchatClient.java:38]: This might be your issue

**Comment by danthedaniel** [comment]
Things got a little wild there for a second with git.

**Pull request merged**